generator/generator_tool/generator_tool.cpp

```diff
@@ -82,6 +83,7 @@ DEFINE_uint64(planet_version, my::SecondsSinceEpoch(),
 DEFINE_string(srtm_path, "",
               "Path to srtm directory. If it is set, generates section with altitude information "
               "about roads.");
+DEFINE_bool(generate_routing_index, false, "Generates section with routing index");
```

Comment:
How this flag is related with FLAGS_routing in https://github.com/mapsme/omim/pull/4567? If they're the same, rename this to routing.